src/Symfony/Component/Cache/CacheItem.php

```diff
@@ -197,3 +197,5 @@ private function unpack(): bool
         return true;
     }
 }
+
+// @php-cs-fixer-ignore protected_to_private This class, while final, exposes protected properties when serialized.
```

Comment:
this class is not means to be serialized ;)
```suggestion
// @php-cs-fixer-ignore protected_to_private Friend-level scope access relies on protected properties
```

Reply:
ack, will apply, thanks !